src/fhirdefs/FHIRDefinitions.ts

```diff
@@ -103,6 +107,43 @@ export class FHIRDefinitions implements Fishable {
     return this.predefinedResources.get(file);
   }
 
+  resetPredefinedResources() {
+    this.predefinedResources = new Map();
+  }
+
+  fishForPredefinedResource(item: string, ...types: Type[]): any | undefined {
+    const resource = this.fishForFHIR(item, ...types);
+    if (
+      resource &&
+      this.allPredefinedResources().find(
+        predefResource =>
+          predefResource.id === resource.id && predefResource.resourceType === resource.resourceType
+      )
+    ) {
+      return resource;
+    }
+  }
+
+  fishForPredefinedResourceMetadata(item: string, ...types: Type[]): any | undefined {
+    const resource = this.fishForFHIR(item, ...types);
+    if (
+      resource &&
+      this.allPredefinedResources().find(
+        predefResource =>
+          predefResource.id === resource.id && predefResource.resourceType === resource.resourceType
+      )
+    ) {
```

Comment:
If you used `this.fishForPredefinedResource` on line 128, then couldn't this just be `if (resource) {`?